libraries/rib-base/src/main/java/com/badoo/ribs/minimal/reactive/Relay.kt

```diff
@@ -0,0 +1,17 @@
+package com.badoo.ribs.minimal.reactive
+
+import com.badoo.ribs.minimal.reactive.Cancellable.Companion.cancellableOf
```

Comment:
Drop `minimal` package part maybe?
Do not think it has any value.

Reply:
I'd like to keep all the reinvented wheels in one place.